benchmark/Benchmark/Htoml.hs

```diff
@@ -0,0 +1,44 @@
+{-# LANGUAGE DeriveAnyClass     #-}
+{-# LANGUAGE DeriveGeneric      #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
According to our style guide, `OPTIONS_GHC` should go before `{-# LANGUAGE #-}` in separate section. See example here:

* https://github.com/kowainik/summoner/blob/f5b12ed01e3f5c607398f443153f4f5746557c50/src/Summoner/Process.hs#L1-L4